packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -785,8 +776,33 @@ def _get_metrics():
         assert r.status_code == 200, 'Received unexpected status code when fetching standalone container metrics'
 
         metrics_response = r.json()
-        metric_keys = [datapoint['name'] for datapoint in metrics_response['datapoints']]
-        assert 'statsd_tester.time.uptime' in metric_keys
+
+        assert 'datapoints' in metrics_response, 'got {}'.format(metrics_response)
+
+        uptime_dp = None
```

Comment:
You could replace this with:

```
uptime_dp = [dp for dp in metrics_response['datapoints'] if dp['name'] == 'statsd_tester.time.uptime][0]
```

But this works and is readable, so I won't open an issue =).